Documentation/gettingstarted.rst

```diff
@@ -1042,39 +1040,37 @@ You can observe the newly created endpoints in Cilium, similar to the following
     33115      Disabled      257        mesos:id=goodclient           f00d::a00:20f:0:815b   10.15.220.6    ready
     64189      Disabled      258        mesos:id=badclient            f00d::a00:20f:0:fabd   10.15.152.27   ready    
 
-Marathon runs the tasks as batch jobs with ``stdout`` available under ``/var/lib/mesos``. To simplify the retrieval of the ``stdout`` log, use the ``tail_client.sh`` script to output each of the client logs. Tail the *goodclient* logs:
+Marathon runs the tasks as batch jobs with ``stdout`` available under ``/var/lib/mesos``. To simplify the retrieval of the ``stdout`` log, use the ``tail_client.sh`` script to output each of the client logs. In a new terminal, go to ``/cilium/examples/mesos``, start a new ssh session to the Vagrant VM with ``vagrant ssh`` and tail the *goodclient* logs:
 
 ::
 
     $ ./tail_client.sh goodclient
 
-and in a separate terminal, start another ssh session to the Vagrant VM with ``vagrant ssh`` and observe the *badclient* logs:
+and in a separate terminal, do the same thing with ``vagrant ssh`` and observe the *badclient* logs:
 
 ::
 
     $ ./tail_client.sh badclient
 
-Make sure the previous command continuously prints the result of the client accessing */public* and */private* API of the web-server:
+Make sure both tail logs continuously prints the result of the clients accessing the */public* and */private* API of the web-server:
 
 ::
 
-    ...
-    --------------------------------
-    GET /public
-    OK
-    
-    HTTP code: 200
-    GET /private
-    OK
-    
-    HTTP code: 200
-    --------------------------------
-    ...
+     ...
+     ---------- Test #X  ----------
+        Request:   GET /public
+        Reply:     OK
+      
+        Request:   GET /private
+        Reply:     OK
+     -------------------------------
+     ...
+
 
 Step 7: Apply L3/L4 Policy with Cilium
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In a new terminal session, do a ``vagrant ssh`` to get into the Vagrant VM and apply an L3/L4 policy only allowing the *goodclient* to access the *web-server*. The L3/L4 json policy looks like:
+Apply an L3/L4 policy only allowing the *goodclient* to access the *web-server*. The L3/L4 json policy looks like:
 
```

Comment:
not critical in this change, but its probably worth a few sentences to describe the basics of the policy.  Ex: This policy is selecting all tasks with label id=web-server.  It is then applying an ingress policy (i.e., limiting what connections other tasks can make the webserver), indicating that only tasks with label id=goodclient can contact the webserver tasks, and only on TCP port 8181.   

There is probably another line or two that talks about how the L7 policy extends the last policy to limit good client to only the /public URL.  

Note: i'm guessing we don't do this in other GSG (e.g., k8s, docker), so may make sense to do that change once across all of the GSGs.

Reply:
We'll need to be consistent across all GSGs. We can update all at the same time.